app/templates/components/settings/email-preferences-section.hbs

```diff
@@ -1 +1,66 @@
-<p>Email preferences</p>
\ No newline at end of file
+<div class="ui grid">
+  <div class="ui row">
+    <div class="ui column eight wide">
+      <h4 class="ui header">
+        {{t 'Email Updates are '}}
+      </h4>
+    </div>
+    <div class="ui column eight wide right aligned">
+      {{ui-checkbox class='toggle checked'}}
+    </div>
+  </div>
+  <div class="ui divider"></div>
+  <div class="ui row">
+    <div class="ui column sixteen wide">
+      <h3 class="ui header">
+        {{t 'Updates about events'}}
+      </h3>
+    </div>
+  </div>
+</div>
+{{#each preferences as |event|}}
+  {{#ui-accordion class='ui segment'}}
+    <div class="ui grid title">
+      <div class="ui column eight wide">
+        <h4 class="ui header">{{event.name}}</h4>
+      </div>
+      <div class="ui column eight wide right aligned">
+        <a class="ui circular label {{if (or event.isNewPaper event.isChangeSchedule event.isNewEvent) 'green' 'yellow'}}">
+          {{#if (or event.isNewPaper event.isChangeSchedule event.isNewEvent)}}
+            {{t 'On'}}
+          {{else}}
+            {{t 'Off'}}
+          {{/if}}
+        </a>
+      </div>
+    </div>
+    <div class="ui grid content">
+      <h4 class="ui header">{{event.role}} {{t 'Notifications'}}</h4>
+      <div class="ui divided"></div>
+      <div class="row">
+        <div class="column eight wide">
+          {{t 'New Paper is Submitted to your Event'}}
+        </div>
+        <div class="ui column eight wide right aligned">
+          {{ui-checkbox class='toggle' checked=event.isNewPaper}}
```

Comment:
@geekyd 

Instead of this use ember check box helper. This doesn't update the value on change on less you use a the attached action.

```hbs
    <div class="ui toggle checkbox">
      {{input type='checkbox' checked=event.isNewPaper}}
    </div>
```

Sorry ... Just noticed then when I was reviewing a different PR